ajax/libs/jquery.webticker/3.0.0/jquery.webticker.js

```diff
@@ -0,0 +1,508 @@
+/*!
+ * webTicker 2.2.0
+ * Examples and documentation at:
+ * http://jonmifsud.com/open-source/jquery/jquery-webticker/
+ * 2011 Jonathan Mifsud
+ * Version: 2.2.0 (15-FEB-2016)
+ * Dual licensed under the Creative Commons and DonationWare licenses:
+ * http://creativecommons.org/licenses/by-nc/3.0/
```

Comment:
Just like other reviewers know: This info is outdated in v3. See https://github.com/mazedigital/Web-Ticker/commit/8b80c4e84495aa35f2f052e77f88d45249ff7e76